k8s/start.sh

```diff
@@ -7,9 +7,21 @@
 # a) set your Scalyr write key in your shell ( e.g. export SCALYR_API_KEY=<YOUR_SCALYR_KEY> )
 # b) edit configmap.yaml to set cluster name (to match the k8s cluster you want to monitor)
 # c) edit configmap.yaml to set `SCALYR_K8S_RATELIMIT_CLUSTER_NUM_AGENTS` to the approximate number of Kubernetes Nodes in your cluster.
-# d) source ./start.sh
+# d) source ./start.sh <configmap_file_name>
 ########################################################################
 
+
+CONFIGMAP_FILE="configmap.yaml"
+
+if [ -n "$1" ]; then
+    CONFIGMAP_FILE=$1
+fi
+
+if [ -z "$SCALYR_API_KEY" ]; then
+    echo "Please set SCALYR_API_KEY environment variable.  Exiting."
+    return
```

Comment:
This should be `exit `

`return` doesn't stop the script, and because it's an error we should exit with a non-zero error code.

Reply:
Hmm, the instructions gave ask them to source the `start.sh` script instead of running directly.  If they are used to that, and they source the script, it will exit their actual shell and they don't even see the error message.  I'm hesitant to change the instructions (from source start.sh -> just ./start.sh.. 

I propose eliminating this check entirely. 

We should get this instruction out to mgmt.

Reply:
(To clarify, it will exit the shell from which they run the command, which would be very frustrating)

Reply:
I sourced the script with the api key not set, and the script it kept on running, and the agent fails to start up.

It needs to be exit, even when sourcing.

Reply:
Correction, I didn't source the shell... :-/

Reply:
Slack?